service/telemetry.go

```diff
@@ -123,6 +136,13 @@ func (tel *telemetryInitializer) initOnce(buildInfo component.BuildInfo, logger
 		telAttrs[semconv.AttributeServiceVersion] = buildInfo.Version
 	}
 
+	if tel.registry.IsEnabled(allowTraceContextPropagationFeatureGateID) {
```

Comment:
This absolutely needs to be documented. Can it be done as part of this PR?

Reply:
Thanks @jpkrohling   I will add the section to telemetry doc in another PR after this is merged.

Reply:
I think a simpler implementation is to make this featuregate just control the default of the configuration. So the list is empty if the gate is not enabled, otherwise is `["tracecontext]`? What do you think? Then be default we don't send this, also we have the gate to change the behavior to be default send, and for people that enable the config works without any gate?